docs/synphot/bandpass.rst

```diff
@@ -269,3 +270,36 @@ set at 0.8:
     from synphot import SpectralElement
     bp = SpectralElement(Const1D, amplitude=0.8)
     bp.plot([1000, 10000], title='Flat Bandpass')
+
+
+.. _synphot-bandpass-specutils:
+
+specutils
+---------
+
+A bandpass can be constructed from and written to `specutils.Spectrum1D`
+object. See :ref:`specutils documentation <specutils:specutils>` for more
+information on how to use `~specutils.Spectrum1D`.
+
+The example below writes a :ref:`synphot-box-bandpass` to a
+`~specutils.Spectrum1D` object:
+
+.. doctest-requires:: specutils
+
+    >>> from astropy import units as u
+    >>> from synphot import SpectralElement
+    >>> from synphot.models import Box1D
+    >>> bp = SpectralElement(Box1D, amplitude=1, x_0=600*u.nm, width=10*u.nm)
+    >>> spec = bp.to_Spectrum1D()
+
+Meanwhile, this example reads in a bandpass from a
+`~specutils.Spectrum1D` object:
+
+.. doctest-requires:: specutils
+
+    >>> from astropy import units as u
+    >>> from specutils import Spectrum1D
+    >>> from synphot import SpectralElement
+    >>> spec = Spectrum1D(spectral_axis=[100, 300]*u.nm,
+    ...                   flux=[0.1, 0.8]*u.dimensionless_unscaled)
```

Comment:
Question which may or may not need changing: what happens if I pass in a spectrum that is *not* `dimensionless_unscaled`?  I don't see anywhere that this is checked for whether or not the unit makes sense.  I'm not sure it needs to be, but it might be a good idea since otherwise the units will get all funky, right?

Reply:
Unit checking is done inside `SpectralElement` and this is already tested somewhere else. You would get this error if you pass in something with dimension (e.g., Jy):
```
UnitConversionError: 'Jy' (spectral flux density) and '' (dimensionless) are not convertible
```